model/_plugins/notifications/update_config/operations.smithy

```diff
@@ -0,0 +1,23 @@
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/observing-your-data/notifications/api/"
+)
+
+@xOperationGroup("notifications.update_config")
+@xVersionAdded("2.2")
```

Comment:
Is this the version of core or the plugin itself?

Reply:
hmm, there relevant notifications page in the documentation only exists as of 2.2.0

But, looking at the [2.0.0 release notes](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.0.0.md#opensearch-notifications) it seems like notifications plugin came with 2.0.0. I'll update it accordingly, let me know if a different value is expected there.